tools/sdk/lib/fix_sdk_libs.sh

```diff
@@ -16,3 +16,10 @@ xtensa-lx106-elf-objcopy --redefine-sym default_hostname=wifi_station_default_ho
 xtensa-lx106-elf-objcopy --redefine-sym default_hostname=wifi_station_default_hostname eagle_lwip_if.o 
 xtensa-lx106-elf-ar r libmain.a eagle_lwip_if.o user_interface.o 
 rm -f eagle_lwip_if.o user_interface.o
+
+# Replace use of ROM ets_intr_(un)lock with nestable ets_intr_(un)lock_nest
+for f in libmain.a libpp.a libnet80211.a; do
```

Comment:
If we're changing every instance of this, the simplest and most direct way, w/o faffing about with the lib.a files (and making sure that it stays stuck when we upgrade SDK revs), this isn't necessary.

Go back to the original versions of these libs and remove the `PROVIDES` line in the `/tools/sdk/ld/*rom.ld` file for `ets_intr_lock` and `ets_intr_unlock` and then it will link against your version directly.

Reply:
As far as I can see it links against functions in libs in preference to rom functions, so all it needs is naming the funcs the same as the original rom ones. Builds without problems and doesn't need modifications to the libs.
I can remove the PROVIDES as well if required, but it works without it.